packages/components/src/components/as-dropdown/as-dropdown.tsx

```diff
@@ -55,6 +55,7 @@ export class Dropdown {
 
   @State() private isOpen: boolean = false;
   @State() private selectedOptionObject: DropdownOption = {};
+  private _docListener: () => void;
```

Comment:
I think we can use closeList as the default value

Reply:
Hmmm I dunno, let me try it. There was a context issue, hence the bind call below.

Reply:
It works, but only if I bind (as expected). `private _docListener: () => void = this._closeList.bind(this)`

So would you propose making this a Prop() users can configure?